docs/learn-staking.md

```diff
@@ -137,10 +144,26 @@ GRANDPA and BABE equivocation slashing penalty is calculated as below:
 Validators may run their nodes on multiple machines to make sure they can still perform validation work in case one of their nodes goes down. It should be noted that if they do not have good coordination to manage signing machines, then equivocation is possible.
 
 > Notice:
-If a validator is reported for any one of the offences they will be removed from the validator set and they will not be paid while they are kicked out.
+If a validator is reported for any one of the offences they will be removed from the validator set ([chilled](#chilling)) and they will not be paid while they are kicked out.
 
 If you want to know more details about slashing, please look at our [research page](https://research.web3.foundation/en/latest/polkadot/slashing/amounts.html).
 
+### Chilling
+
+Chilling is the act of removing a validator from the active validator set, also disqualifying them from the set of electable candidates in the next NPoS cycle. This carries an implied penalty of being unable to earn rewards for the duration of the next era, and requires them to earn back any nominations they may have had before being removed.
```

Comment:
We might want to add that this is not necessarily a penalty - a validator may choose to self-chill because, e.g., they know that a hurricane is coming which may disrupt their network connection.

Reply:
Good thinking